### PR TITLE
master | LRQA-42529

### PR DESCRIPTION
**Pull request by Hanlf**
https://issues.liferay.com/browse/LRQA-42529
 
cc/ @brianwulbern

### CODE REVIEW

**Comment by Hanlf** [comment]
ci:test:sf

**Comment by brianwulbern** [comment]
@Hanlf 
```
  [exec] timeout.implicit.wait=3
     [exec] 
     [exec] Exception in thread "main" java.lang.Exception: Error on line 1 of document  : Content is not allowed in prolog. Nested exception: Content is not allowed in prolog.
     [exec] Invalid syntax in /opt/dev/projects/github/liferay-portal/portal-web/test/functional/com/liferay/portalweb/tests/enduser/collaboration/blogs/PGBlogs.testcase
     [exec] 	at com.liferay.poshi.runner.PoshiRunnerGetterUtil.getRootElementFromURL(PoshiRunnerGetterUtil.java:511)
     [exec] 	at com.liferay.poshi.runner.PoshiRunnerGetterUtil.getRootElementFromURL(PoshiRunnerGetterUtil.java:386)
     [exec] 	at com.liferay.poshi.runner.PoshiRunnerContext._storeRootElements(PoshiRunnerContext.java:1423)
     [exec] 	at com.liferay.poshi.runner.PoshiRunnerContext._readPoshiFiles(PoshiRunnerContext.java:1050)
     [exec] 	at com.liferay.poshi.runner.PoshiRunnerContext._readPoshiFiles(PoshiRunnerContext.java:1024)
     [exec] 	at com.liferay.poshi.runner.PoshiRunnerContext.readFiles(PoshiRunnerContext.java:297)
     [exec] 	at com.liferay.poshi.runner.PoshiRunnerContext.readFiles(PoshiRunnerContext.java:293)
     [exec] 	at com.liferay.poshi.runner.PoshiRunnerContext.main(PoshiRunnerContext.java:283)
     [exec] Caused by: org.dom4j.DocumentException: Error on line 1 of document  : Content is not allowed in prolog. Nested exception: Content is not allowed in prolog.
     [exec] 	at org.dom4j.io.SAXReader.read(SAXReader.java:482)
     [exec] 	at org.dom4j.io.SAXReader.read(SAXReader.java:343)
     [exec] 	at com.liferay.poshi.runner.PoshiRunnerGetterUtil.getRootElementFromURL(PoshiRunnerGetterUtil.java:507)
     [exec] 	... 7 more
     [exec] Data loss has occurred while parsing Poshi Script at:
     [exec] /opt/dev/projects/github/liferay-portal/portal-web/test/functional/com/liferay/portalweb/tests/enduser/collaboration/blogs/PGBlogs.testcase:1
     [exec] @component-name = "portal-collaboration"
```

**Comment by GinsonRen** [comment]
No test changes needed :)

**Comment by GinsonRen** [comment]
Pushed to AChiang.
https://github.com/austinchiang/liferay-portal/pull/334

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/collaboration/blogs/PGBlogs.testcase [hunk 1]
```diff
@@ -1393,6 +1393,24 @@ definition {
...
 			subject = "Blogs Entry Title"
 		);
+
+		Navigator.gotoPage(pageName = "Blogs Page");
+
+		Blogs.editEntryPG(
+			entryContent = "Blogs Entry Content",
+			entryContentEdit = "Blogs Entry Content Edit",
+			entryTitle = "Blogs Entry Title",
+			entryTitleEdit = "Blogs Entry Title Edit"
```
Comment (jpince) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/collaboration/blogs/PGBlogs.testcase:1403]: @Hanlf you are missing a comma at the end of this line

Comment (Hanlf) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/collaboration/blogs/PGBlogs.testcase:1403]: Apologize for the inconvenience caused, thank you @jpince for catching it, sorry!

Comment (jpince) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/collaboration/blogs/PGBlogs.testcase:1403]: @Hanlf no problem, we are all still learning the new POSHI Script :)

**Pull request closed without merging**